packages/plugins/src/SidebarPlugin.ts

```diff
@@ -172,14 +152,18 @@ const SidebarPlugin: PluginType<SidebarPluginPage, SidebarPluginOptions, Sidebar
         deep: sidebarRootLevel
       });
 
-      //Map into Sidebar Groups and sort children according to priority, priority = 1 has the highest priority
+      //Map into Sidebar Groups and sort children according to priority
       function sortSidebarGroups(sidebarData) {
         const sortedGroupedPages = sidebarData.map(page => {
-          if (page.childNodes) {
-            const sortedPages = page.childNodes.sort((a, b) => b.priority - a.priority);
+          if (page.childNodes.length > 1) {
+            const sortedPages = page.childNodes.sort(
+              (pageA, pageB) =>
```

Comment:
generally nicer to have a named sort function as this is then not an anonymous sort..
See how we named it sortByPathLevel or sortPagesByPriority

I don't have to read the code to know how the sort works.